File: C Modules/Assignments/Assignment no 8/question-3.c
#include<stdio.h>
int main()
{
    int n,size,sum=0;


    printf("Enter the Size of array:");
    scanf("%d",&size);

    printf("Enter the number of elements:");
    scanf("%d",&n);

    int arr[size];

    printf("Enter the number:\n");

    for (int i = 0; i < n; i++)
    {
        scanf("%d",&arr[i]);
    }

    for (int i = 0; i < n; i++)
    {
        sum=sum+arr[i];
        
    }
    printf("sum of all numbers is %d",sum);
        
    return 0;

}

File: Java Module/Assignments in c to java/Assignment-03/Question_3.java
public class Question_3 {

    public static void main(String[] args) {
            int sum=0,end=10,start=1;
    // printf("Enter the start value:");
    // scanf(" %d",&start);
    
    // printf("Enter the end value:");
    // scanf(" %d",&end);
    while (start<=end)
    {
        sum=sum+start;
        start++;
    }
    System.out.printf("output: %d",sum);
    }
}